dokka-subprojects/analysis-kotlin-descriptors/build.gradle.kts

```diff
@@ -47,5 +47,34 @@ tasks.shadowJar {
  *   and so such classes are not replaced afterward by `shadowJar` task - it visits every class once
  *
  */
-dependencies.shadow(libs.fastutil)
-tasks.shadowJar { exclude("it/unimi/dsi/fastutil/**") }
+
+// configuration for dependencies which we need to replace with original ones because `kotlin-compiler` minimizes them
+val shadowOverride: Configuration by configurations.creating {
```

Comment:
```suggestion
val shadowOverride: Configuration by configurations.creating {
    description = "dependencies which we need to replace with original ones because `kotlin-compiler` minimizes them"
```